internal/core/items/component_container.rs

```diff
@@ -219,10 +220,21 @@ impl Item for ComponentContainer {
 
     fn render(
         self: Pin<&Self>,
-        _backend: &mut super::ItemRendererRef,
-        _item_rc: &ItemRc,
-        _size: LogicalSize,
+        backend: &mut super::ItemRendererRef,
+        item_rc: &ItemRc,
+        size: LogicalSize,
     ) -> RenderingResult {
+        if let Some(item_tree) = self.item_tree.borrow().clone() {
+            let item_tree = vtable::VRc::borrow_pin(&item_tree);
+            let root_item = item_tree.as_ref().get_item_ref(0);
+            let window_item =
+                crate::items::ItemRef::downcast_pin::<crate::items::WindowItem>(root_item).unwrap();
```

Comment:
I think the `unwrap` code here is dangerous.

I know that the compiler currently ensure that there is a WindowItem always, but i don't think it will be the case in general in the future (when we want, eg, Button, to be an embedded component)

If there is no window_item, we just don't need to draw a background.